src/style.css

```
@@ -0,0 +1,47 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  width: 100px;
+  margin: 0 auto;
+  margin-top: 50px;
+}
+
+.stars--0,
+.stars--1, 
+.stars--2,
+.stars--3,
+.stars--4,
+.stars--5 {
```

```suggestion
.stars {
```
Этот класс есть у всех блоков